src/lib/utils/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as fakeAudioStream } from "./fakeAudioStream";
 export { default as fakeWebcamFrame } from "./fakeWebcamFrame";
+export { createStore as createMockStore } from "../core/redux/tests/store.setup";
```

Comment:
This now becomes part of public API for browser-sdk, which means whenever we chnage it we need to make sure it's not API breaking change. I don't think that is a good idea, I'd reather have that code duplicated in the node-agent TBH.